### PR TITLE
doc: openssl and zlib assessment

### PR DESCRIPTION
**Pull request by RafaelGSS**
https://github.com/nodejs-private/node-private/issues/356

cc: @nodejs/security

### CODE REVIEW

**Comment by RafaelGSS** [comment]
I took advantage of this assessment and mentioned the Node.js Assessment Workflow. It's fairly important to mention such an initiative when dealing with dependency assessments.

**Review by richardlau — APPROVED** [review]

File: locale/en/blog/vulnerability/openssl-and-zlib-vulnerability-assessment.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+date: 2022-10-24T17:00:15.000Z
```
Comment (richardlau) [inline locale/en/blog/vulnerability/openssl-and-zlib-vulnerability-assessment.md]: Note that the server runs on UTC -- at the time I'm writing this comment this time is ten minutes in the future 🙂.

Comment (RafaelGSS) [reply locale/en/blog/vulnerability/openssl-and-zlib-vulnerability-assessment.md]: Increased 3 hours

Comment (richardlau) [reply locale/en/blog/vulnerability/openssl-and-zlib-vulnerability-assessment.md]: I was thinking about whether a date in the future would prevent the page showing up when the website is rebuilt, but maybe that only applies to the banner and the date for blog posts only affects the ordering in which they are listed?

Comment (RafaelGSS) [reply locale/en/blog/vulnerability/openssl-and-zlib-vulnerability-assessment.md]: Does a PR gets merged with a date in the past? That's the usual workflow?

Comment (richardlau) [reply locale/en/blog/vulnerability/openssl-and-zlib-vulnerability-assessment.md]: Let's not worry about it for now -- we can always fix it up later if it's an issue.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by ruyadorno — APPROVED** [review]

**Pull request merged**